src/urdf/UrdfMaterial.js

```diff
@@ -34,4 +34,14 @@ function UrdfMaterial(options) {
   }
 }
 
-module.exports = UrdfMaterial;
\ No newline at end of file
+UrdfMaterial.prototype.isLink = function() {
+  return this.color === null && this.textureFilename === null;
+};
+
+UrdfMaterial.prototype.assign = function(obj) {
+  for( var key in obj ) {
```

Comment:
This can just be

```js
var assign = require("object-assign");

UrdfMaterial.prototype.assign = function(obj) {
  return assign(this, obj);
}
```